### ISSUE TITLE
More flexibility for theme settings - in particular when Color Module is enabled

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#6750]
## Description of the need

If a theme uses the Color module, all settings are forced into the left column while the preview for the color module takes up 2/3 of the page.

![image](https://github.com/user-attachments/assets/f2a7f0c7-0d92-4c65-be8f-f11580db714f)

We have situations in which this is very cramped and unpractical. See: https://github.com/backdrop-contrib/css_editor/issues/6

Or see the CSS options in Tatsu:
https://github.com/backdrop-contrib/tatsu/issues/72

![image](https://github.com/user-attachments/assets/82b7d23e-a59d-4af9-bd8f-05001c43a306)

I've been told that there is no current option in core to put theme settings anyplace other than this one column. 

## Proposed solution

I would like the option to place a theme setting above or below the page preview such that it can stretch the full width of the page OR the ability to add another tab or another settings page to a theme. 

I've been told that this is currently not possible.

**Comment by stpaultim** [comment]
This is what the CSS editor looks like in Bootstrap 5 lite, without the color module enabled:
![image](https://github.com/user-attachments/assets/28dd8b9e-497c-41e3-b5a7-37a1c403fff8)

This is what it looks like in Lateral:
![image](https://github.com/user-attachments/assets/9f1168db-9f54-48ce-968e-3b99ac1e9833)

**Comment by theflightrs** [comment]
Agree, this _needs_ to be full width. In addition to that i'd even suggest getting the full box as some sort of modal popup on the frontend (e.g. from the side, like an inline iFrame?) For more fluidity.
